### DISCUSSION TITLE
I subscribed to github copilot and I want to unsubscribe and return my $100. What should I do

### DISCUSSION THREAD

**Discussion by imxiny** [community/community#30479] [Copilot Conversations] [answered]
I subscribed to github copilot and I want to unsubscribe and return my $100. What should I do

**Comment by strugglebak** [comment]
report it to Github support

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by Amltoni** [comment]
This group added this to steal my investment. Please put a permanent ban on my name.
